.github/workflows/Dockerfile.buster

```diff
@@ -2,19 +2,23 @@ FROM debian:buster
 ARG DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends apt-utils ca-certificates gnupg wget && \
+    apt-get install -y --no-install-recommends apt-utils ca-certificates gnupg wget lsb-release && \
     echo "deb https://deb.debian.org/debian buster-backports main" >> /etc/apt/sources.list && \ 
     wget -qO - https://repo.vkpartner.ru/GPG-KEY.pub | apt-key add - && \
     echo "deb https://repo.vkpartner.ru/kphp-buster/ buster main" >> /etc/apt/sources.list && \
     wget -qO - https://packages.sury.org/php/apt.gpg | apt-key add - && \
     echo "deb https://packages.sury.org/php/ buster main" >> /etc/apt/sources.list.d/php.list && \
+    TEMP_DEB="$(mktemp)" && \
+    wget -O "$TEMP_DEB" 'https://dev.mysql.com/get/mysql-apt-config_0.8.22-1_all.deb' && \
+    dpkg -i "$TEMP_DEB" && \
+    rm -f "$TEMP_DEB" && \
```

Comment:
just in case - do we need this? If yes, there is no other way to obtain package?